### ISSUE TITLE
External Links hide behind the website 301 redirect could not open in browser

### ISSUE THREAD

**Issue by hye** [sindresorhus/Plash#79]
**Describe the bug**

External Links hide behind the website redirect could not open in browser

**To reproduce**

1. website is https://www.producthunt.com/
2. browsing mode
3. click either one project `GET IT` ,eg  the link is `https://www.producthunt.com/r/611230e242e086`, the website will response 301 redirect `https://www.redacted.com/product/soft-ui-dashboard?ref=producthunt`, but the page still showed in Plash windows

**Expected behavior**

Detect 301 http code? then open it in System default browser?

**Info**

 - macOS version: 10.15.7
 - Plash version:  Version 2.1.0 (16)

**Additional context**

#73

**Comment by sindresorhus** [comment]
> Detect 301 http code? then open it in System default browser?

I agree, but it's not as easy as it sounds as there are a lot of moving parts. I'll try to look closer into this soon.

**Comment by hye** [comment]
https://developer.apple.com/documentation/foundation/urlsessiontaskdelegate/1411626-urlsession
